doc/enterprise/auth/auth.md

```diff
@@ -0,0 +1,234 @@
+you have activated Pachyderm Enterprise Edition
+as described in [this guide](./deployment.html).
+
+To enable access controls, complete the following steps:
+
+1. Verify the status of the Enterprise
+features by opening the Pachyderm dashboard in your browser or
+by running the following `pachctl` command:
+
+   ```
+   $ pachctl enterprise get-state
+   ACTIVE
+   ```
+
+1. Activate the Enterprise access control features by completing
+the steps in one of these sections:
+
+   * [Activating Access Control in the Dashboard](#activating-access-controls-with-the-dashboard)
+   * [Activating Access Control in pachctl](#activating-access-controls-with-pachctl)
+
+### Activating access controls with the dashboard
+
+To activate access controls in the Pachyderm dashboard,
+complete the following steps:
+
+1. Go to the Settings page.
+1. Click the **Activate Access Controls** button.
+After you click the button, Pachyderm enables you to add GitHub users
+as cluster admins and activate access control:
+
+![alt tag](auth_dash1.png)
+
+After activating access controls, you should see the following screen
+that asks you to log in to Pachyderm:
+
+![alt tag](auth_dash2.png)
+
+### Activating access controls by using `pachctl`
+
+To activate access controls by using `pachctl`, run the following
+command:
+
+```
+$ pachctl auth activate --admins=<user>
+```
+
+If you want the Pachyderm cluster to have more than one admin,
+specify them as a comma-separated list.
+
+## Logging in to Pachyderm
+
+After you activate access control, log in to your cluster either
+through the dashboard or CLI. The CLI and the dashboard have
+independent login workflows:
+
+- [Log in to the dashboard](#login-on-the-dashboard).
+- [Log in by using the CLI](#login-using-pachctl).
+
+### Log in to the dashboard
+
+After you have authorized access controls for Pachyderm, you
+need to log in to use the Pachyderm dashboard as shown above
+in [this section](#activating-access-controls-with-the-dashboard).
+
+To log in to the dashboard, complete the following steps:
+
+1. Click the **Get GitHub token** button. If you
+have not previously authorized Pachyderm on GitHub, an option
+to **Authorize Pachyderm** appears. After you authorize
+Pachyderm, a Pachyderm user token appears:
+
+![alt tag](auth.png)
+
+1. Copy and paste this token back into the Pachyderm login
+screen and press **Enter**. You are now logged in to Pachyderm,
+and you must see your GitHub avatar and an indication of your
+user in the upper left-hand corner of the dashboard:
+
+![alt tag](auth_dash3.png)
+
+
+### Log in to the CLI
+
+To authenticate by using `pachctl`, run the following
+command:
+
+1. Log in with your user name:
+
+   ```bash
+   pachctl auth login <username>
+   ```
+
+   When you run this command, `pachctl` provides
+   you with a GitHub link to authenticate as the provided
+   GitHub user.
+
+   **Example:**
+
+   ```bash
+   $ pachctl auth login dwhitena
```

Comment:
Similar to above—login shouldn't have a `username` argument. In this case, for example, `dwhitena` would be ignored and the user would log in as whoever they were on GitHub